aws/sdk/build.gradle.kts

```diff
@@ -279,7 +279,6 @@ fun rewritePathDependency(line: String): String {
 }
 
 tasks.register<Copy>("copyAllRuntimes") {
-    dependsOn("smithyBuildJar")
```

Comment:
Commenting for posterity: this makes the relocate runtime tasks run without regenerating the entire SDK.

Reply:
:-) thanks for fixing